mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -92,7 +92,7 @@ public QuotaUsageUpdatedEvent(MailboxSession.SessionId sessionId, User user, Quo
 
         public QuotaUsageUpdatedEvent(User user, QuotaRoot quotaRoot,
                                       Quota<QuotaCount> countQuota, Quota<QuotaSize> sizeQuota, Instant instant) {
-            this(MailboxSession.SessionId.zero(), user, quotaRoot, countQuota, sizeQuota, instant);
+            this(MailboxSession.SessionId.random(), user, quotaRoot, countQuota, sizeQuota, instant);
```

Comment:
random because you don't have one ? really ?

Reply:
updated Optional in https://github.com/linagora/james-project/pull/1980/commits/7b2709baa1cde1bb164fc746cd64c0bb2510fc32